tests/pillar/cluster.sls

```diff
@@ -4,6 +4,12 @@ horizon:
     version: liberty
     secret_key: secret
     session_timeout: 43200
+    plugin:
```

Comment:
This will work only on Ubuntu :-/ I don't think we want to have this pillar as tests should be universal (or split across distros).